src/app/notifications/notifications.component.ts

```diff
@@ -78,4 +78,11 @@ export class NotificationsComponent implements OnInit, AfterViewInit {
       }, (err) => console.log(err));
     }
   }
+  readAllNotification() {
+    if (!this.emptyData) {
+      this.notifications.data.forEach(element => {
```

Comment:
This will perform an HTTP request for each unread notification to update its status.

You can use the `bulkDocs` method on couchService.  The final parameter should be an array of the "unread" notification with their `status` property changed to `read`.